test/type/varlen_pool_test.cpp

```diff
@@ -1,391 +0,0 @@
-//===----------------------------------------------------------------------===//
-//
-//                         Peloton
-//
-// varlen_pool_test.cpp
-//
-// Identification: test/common/varlen_pool_test.cpp
-//
-// Copyright (c) 2015-16, Carnegie Mellon University Database Group
-//
-//===----------------------------------------------------------------------===//
-
-#include <limits.h>
-#include <pthread.h>
-#include "type/varlen_pool.h"
-#include "gtest/gtest.h"
-#include "common/harness.h"
-
-namespace peloton {
-namespace test {
-
-class VarlenPoolTests : public PelotonTest {};
```

Comment:
Why not keep this for the new `EphemeralPool`? The API should be the same, right?